lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -73,6 +70,52 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * List the import conditions found in the prelude of an `@import` rule
+ *
+ * @param {Node[]} params
+ * @typedef {import('postcss-value-parser').Node} Node
```

Comment:
`@template` can reduce `Node` repetition like this. But it's OK to leave it as-is since it's trivial. 👌🏼 

```suggestion
 * @template {import('postcss-value-parser').Node} T
 * @param {T[]} params
```